static/js/util.js

```diff
@@ -69,6 +69,14 @@ exports.is_pm_recipient = function (email, message) {
     return recipients.indexOf(email.toLowerCase()) !== -1;
 };
 
+exports.extract_pm_recipients = function (recipients) {
+    if (recipients) {
+        return recipients.split(/\s*[,;]\s*/);
+    } else {
+        return compose.recipient().split(/\s*[,;]\s*/);
+    }
+};
```

Comment:
I think the API would be cleaner if it always required an argument and the compose code just passed `compose.recipient()` to this funciton